browser/app/js/components/Login.js

```diff
@@ -14,115 +14,118 @@
  * limitations under the License.
  */
 
-import React from 'react'
-import classNames from 'classnames'
-import logo from '../../img/logo.svg'
-import Alert from 'react-bootstrap/lib/Alert'
-import * as actions from '../actions'
-import InputGroup from '../components/InputGroup'
+import React from "react"
+import { connect } from "react-redux"
+import classNames from "classnames"
+import logo from "../../img/logo.svg"
+import Alert from "./Alert"
+import * as actionsAlert from "../actions/alert"
+import InputGroup from "../components/InputGroup"
+import { minioBrowserPrefix } from "../constants"
+import web from "../web"
+import { Redirect } from "react-router-dom"
 
-export default class Login extends React.Component {
+export class Login extends React.Component {
   handleSubmit(event) {
     event.preventDefault()
-    const {web, dispatch, loginRedirectPath} = this.props
-    let message = ''
-    if (!document.getElementById('accessKey').value) {
-      message = 'Secret Key cannot be empty'
+    const { dispatch, history } = this.props
+    let message = ""
+    if (!document.getElementById("accessKey").value) {
+      message = "Secret Key cannot be empty"
     }
-    if (!document.getElementById('secretKey').value) {
-      message = 'Access Key cannot be empty'
+    if (!document.getElementById("secretKey").value) {
+      message = "Access Key cannot be empty"
```

Comment:
@kmkanagaraj, similar to my previous comment, just noticed that the message here should be "Secret Key cannot be empty", because we are checking for the secretKey here